requirements/constraints.txt

```
@@ -0,0 +1,14 @@
+# Version constraints for pip installation.
+#
+# This file doesn't install any packages. It specifies version constraints
+# that will be applied if a package is needed.
+#
+# When pinning something here, please provide an explanation of why.  Ideally,
+# link to other information that will help people in the future to remove the
+# pin when possible.  Writing an issue against the offending project and
+# linking to it here is good.
+
+
+django<4.0
```

Constraint should be `django<3.3` instead.
```suggestion
django<3.3
```